deploy/charts/secret-manager/README.md

```diff
@@ -0,0 +1,74 @@
+# secret-manager
+
+![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![Version: 0.2.2](https://img.shields.io/badge/Version-0.2.2-informational?style=flat-square) ![AppVersion: %%APP_VERSION%%](https://img.shields.io/badge/AppVersion-%%APP_VERSION%%-informational?style=flat-square)
```

Comment:
It looks like the badge and Readme instructions on main branch will be broken due to the way the release updates this version. Not sure if there is a clean way to fix this.